libcodechecker/server/api/report_server.py

```diff
@@ -1936,6 +1936,14 @@ def massStoreRun(self, name, tag, version, b64zip, force):
 
         session = self.__storage_session.get_transaction(run_id)
 
+        all_reports = session.query(Report) \
+            .filter(Report.run_id == run_id) \
+            .all()
+
+        hash_map_reports = defaultdict(list)
```

Comment:
Is `defaultdict` known?

Maybe having a regular dict and
`hash_map_reports[report.bug_id] = [report]`

Is easier to read?

Reply:
We use `defaultdict` in multiple places. This way if a key doesn't exists in a map, it will default initialize it.
```python
hash_map_reports = {}
if report.bug_id not in hash_map_reports:
    hash_map_reports[report.bug_id] = [report]
else:
    hash_map_reports[report.bug_id].append(report)
```

Reply:
Oh, indeed.